phpBB/phpbb/install/installer.php

```diff
@@ -13,6 +13,7 @@
 
 namespace phpbb\install;
 
+use GuzzleHttp\Message\Request;
```

Comment:
You're adding new `use` statements but you're not changing anything in this file. Please remove any unnecessary changes.